### PR TITLE
Update the Setting Up a Gift Registry columns

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request updates the Setting Up a Gift Registry columns

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/marketing/gift-registry-create.html?itm_source=merchdocs&itm_medium=search_page&itm_campaign=federated_search&itm_term=Setting%20Up%20a%20Gift%20Registry

### CODE REVIEW

File: src/marketing/gift-registry-create.md [hunk 1]
```diff
@@ -112,14 +112,14 @@ Select the type of data and input control that is associated with the attribute.
...
 |Event Location|The location of the event that is related to the gift registry. <br/>Set **Is Searcheable** to: Yes / No <br/>Set **Is Listed** to: Yes / No|
-|Role|The role that identifies who the gift is for. <br/>Set **Is Searcheable** to: Yes / No <br/>Set **Is Listed** to: Yes / No <br/>For example, `Bride`, `Groom`, or `Partner`.|
+|Role|The role that identifies who the gift is for. <br/>Set **Is Searcheable** to: Yes / No <br/>Set **Is Listed** to: Yes / No <br/>Click **Add New Option** to add more conditions to the drop-down menu:<br/>**Code** - A unique name to identify the attribute internally.<br/>**Label** - The name that identifies the attribute in the customer’s account dashboard.<br/>**Is Default** - Set the switch to select the default condition.<br/>**Delete Option** - To delete the option, click **Delete Option**.<br/>For example, `Bride`, `Groom`, or `Partner`.|
```
Comment (leslietilling) [inline src/marketing/gift-registry-create.md:122]:
```suggestion
|Role|The role that identifies the gift recipient. For example, `Bride`, `Groom`, or `Partner`.<br/>**Is Searcheable** - Set to `Yes `/ `No` <br/>**Is Listed** - Set to `Yes` / `No` <br/>**Add New Option** - Click  to add more conditions to the drop-down menu:<br/>**Code** - A unique name to identify the attribute internally.<br/>**Label** - The name that identifies the attribute in the customer’s account dashboard.<br/>**Is Default** - Set this switch to select the default condition.<br/>**Delete Option** - Click to delete the option.|
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Just some edits for clarity and formatting.

File: src/marketing/gift-registry-create.md (hunk 1 above)
Comment (ZhovnirMaria) [reply src/marketing/gift-registry-create.md]: @leslietilling Thank you, done  ☑️

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**